cmd/tink-cli/cmd/hardware/get.go

```diff
@@ -0,0 +1,62 @@
+package hardware
+
+import (
+	"context"
+	"io"
+
+	"github.com/jedib0t/go-pretty/table"
+	"github.com/tinkerbell/tink/client"
+	"github.com/tinkerbell/tink/cmd/tink-cli/cmd/get"
+	"github.com/tinkerbell/tink/protos/hardware"
+)
+
+type GetHardware struct {
+	get.CmdOpt
+	metaClient *client.MetaClient
+}
+
+func NewGetHardwareOpt(metaClient *client.MetaClient) GetHardware {
```

Comment:
```suggestion
func NewGetHardware(metaClient *client.MetaClient) GetHardware {
```
So it matches `New$type(...) $type {` pattern